test/integration/server/inspec/server_spec.rb

```diff
@@ -44,6 +44,13 @@
   its('mysqldump.password') { should eq '\'jzYY0cQUnPAMcqvIxYaC\'' }
 end
 
+describe ini('/etc/my.cnf') do
```

Comment:
Looks like they have a [mysql_conf](https://www.inspec.io/docs/reference/resources/mysql_conf/) resource, mind giving that a try?